### PR TITLE
Cleanup Training Templates

### PR DESCRIPTION
**Pull request by superryeti**
There are two identical template files in physionet-django/project/templates/project:

published_project_required_training.html
published_project_required_trainings.html

It looks like the plural one was used in the code, However the singular one looked like a better name. So I removed the plural one and changed code to use the singular named template

### CODE REVIEW

File: physionet-django/project/views.py [hunk 1]
```diff
@@ -1732,15 +1732,15 @@ def published_project_license(request, project_slug, version):
...
     """Displays a published project's required training"""
     project = get_object_or_404(PublishedProject, slug=project_slug, version=version)
 
     required_trainings = project.required_trainings.all()
 
     return render(
         request,
-        'project/published_project_required_trainings.html',
+        'project/published_project_required_training.html',
         {'project': project, 'required_trainings': required_trainings},
```
Comment (tompollard) [inline physionet-django/project/views.py]: I think this line needs updating to match the new template?

`{'project': project, 'required_trainings': required_trainings}` should be:
 `{'project': project, 'required_training': required_trainings}`

**Review by tompollard — COMMENTED** [review]
Thanks @amitupreti! It looks like there is a slight difference between the templates (`{% for training in required_training %}` vs `{% for training in required_trainings %}`). Other than that, looks good to me.

File: physionet-django/project/views.py (hunk 1 above)
Comment (superryeti) [reply physionet-django/project/views.py]: Thanks Tom. I will make the changes

Good Catch

**Comment by tompollard** [comment]
Thanks Amit, looks good to me!

**Pull request merged**